spec/models/comment_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before(:all) do
+    @user = User.new(name: 'John', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @post = Post.new(title: 'My first post', text: 'This is my first post.', author: @user, comments_counter: 4,
+                     likes_counter: 6)
+    let(:comment) do
+      Comment.new(users: @user, posts: @post, text: 'Hey!, it is my first comment')
+    end
+  end
+
+  it 'is not valid without a text' do
```

Comment:
- [ ] It would be better if we make all the test cases pass and marked green :100: Let us kindly fix out model tests(Unit testing)
![Screenshot from 2022-11-25 12-51-17](https://user-images.githubusercontent.com/46388113/203990126-997cf7c3-63f5-4bc9-90b3-3662b046f215.png)